README.md

```diff
@@ -1,19 +1,18 @@
 # gluon-mesh-vpn-wireguard
 
-You can use this package for connecting with wireguard to the freifunk Königswinter network.
+You can use this package for connecting with wireguard to the Freifunk Munich network.
 
 You should use something like the following in the site.conf:
 
         
 ```
  mesh_vpn = {
-        mtu = 1420,
+        mtu = 1410,
         wireguard = {
                 enabled = 'true',
                 iface = 'mesh-vpn',
-                iprange = '10.3', --must be a /16!!
                 limit = '1', -- actually unused
-                gretapip = '10.3.0.2',
+                mcastgroup = 'ff02::15c',
```

Comment:
Great idea to recycle Gluons VXLAN address. Much better than doing GRETAP.

Can you explain how the two-way communication via multicasting will work on the link? Especially with regards to AllowedIPs.

Reply:
It uses multicast flooding to discover the real source addresses (link-local in our case). Which are then used for direct communication. 

https://vincent.bernat.ch/en/blog/2017-vxlan-linux